src/NuGetTrends.Data/NuGetTrendsContext.cs

```diff
@@ -19,10 +19,10 @@ public NuGetTrendsContext(DbContextOptions<NuGetTrendsContext> options)
             : base(options)
         { }
 
-        public DbSet<PackageDetailsCatalogLeaf> PackageDetailsCatalogLeafs { get; set; }
-        public DbSet<Cursor> Cursors { get; set; }
-        public DbSet<DailyDownload> DailyDownloads { get; set; }
-        public DbSet<PackageDownload> PackageDownloads { get; set; }
+        public DbSet<PackageDetailsCatalogLeaf> PackageDetailsCatalogLeafs { get; set; } = null!;
+        public DbSet<Cursor> Cursors { get; set; } = null!;
```

Comment:
Maybe we can use the same approach on the `Id's` properties where we are sure they are not null?

Reply:
agreed